src/main/java/frc/robot/subsystems/PivotIO.java

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import org.littletonrobotics.junction.AutoLog;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public interface PivotIO {
+  @AutoLog
+  public static class PivotIOInputs {
+    public Rotation2d position = new Rotation2d();
+
+    public double motorLeftAppliedVolts = 0.0;
+    public double motorLeftCurrentAmps = 0.0;
+
+    public double motorRightAppliedVolts = 0.0;
+    public double motorRightCurrentAmps = 0.0;
+  }
+
+  public default void updateInputs(PivotIOInputs inputs) {}
+
+  /**
+   * Rotate both motors.
+   * @param speed The speed at which to rotate the motors.
+   */
+  public default void setSpeed(double speed) {}
+
+  /**
+   * Stop both motors.
+   */
+  public default void stopPivot() {}
+
+  /**
+   * Get the position in radians.
+   */
+  public double getPositionRadians();
```

Comment:
Store this as a constant inside PivotIOInputs and update it from the hardware layer. Then you can access it from the subsystem layer by calling inputs.armPositionRadians or whatever.